application/controllers/ConfigController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Icinga\Module\Jira\Controllers;
+
+use GuzzleHttp\Psr7\ServerRequest;
+use Icinga\Module\Jira\Forms\Config\DeploymentConfigForm;
+use Icinga\Module\Jira\Web\Controller;
+use Icinga\Web\Widget\Tab;
+use Icinga\Web\Widget\Tabs;
+
+class ConfigController extends Controller
+{
+    public function init()
+    {
+        $this->assertPermission('config/modules');
+        parent::init();
+    }
+
+    public function deploymentAction()
+    {
+        $this->addTitle('Configure Jira Software Deployment');
+        $form = (new DeploymentConfigForm())->handleRequest(ServerRequest::fromGlobals());
```

Comment:
Please use `$this->getServerRequest()` instead of `ServerRequest::fromGlobals()`

Reply:
Once you did this. Please raise the version requirement of ipl to v0.11.0 in module.info and the documentation.